spec/api/email_listicle_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+describe "email_listicle_api", type: :api do
+  let(:v1_prefix) { "/api/v1" }
+
+  def jsonify_response(response_body)
+    JSON.parse(response_body)
+  end
+  describe "GET /all" do
+    context "when no links to return" do
+      it "returns and empty array for no links " do
+        #setup
+        allow(EmailLink).to receive(:undecided).and_return([])
```

Comment:
is there a reason to use `allow` here instead of `expect` ? Does `EmailLink` not always get called with `#undecided`?